lib/internal/modules/esm/deprecate_process_buffer.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { cjsContext } = require('internal/bootstrap/loaders');
+const { getStackSourceName } = internalBinding('contextify');
```

Comment:
whatever `getStackSourceName` is, it doesn't exist.

Reply:
I believe it does, but please enlighten me further.

Reply:
![](https://gc.gy/d1773669-7185-43b7-9427-cb75e9665677.png)